src/reviewers/review-utils.ts

```diff
@@ -19,3 +30,71 @@ export function calculateOverallConclusionCode(codes: ConclusionCode[]): Conclus
   }
   return ConclusionCode.UNKNOWN;
 }
+
+export function generateAlignmentPairs(config: Configuration, fisher: Fisher): FHITAlignmentPair[] {
+  return config.alignments
+    .map((alignment: FHITAlignment) => {
+      const [sender, receiver] = Object.entries(alignment);
+      const [senderAlias, senderId] = sender;
+      const [receiverAlias, receiverId] = receiver;
+
+      const senderSD = fisher.fishForFHIR(senderId);
+      const receiverSD = fisher.fishForFHIR(receiverId);
+
+      if (!(senderSD && receiverSD)) {
+        if ((senderId === 'NONE' && receiverSD) || (receiverId === 'NONE' && senderSD)) {
+          return {
+            sender: senderSD,
+            receiver: receiverSD
+          };
+        }
+        const absentAliases = [];
+        const absentIDs = [];
+        if (!senderSD) {
+          absentAliases.push(senderAlias);
+          absentIDs.push(senderId);
+        }
+        if (!receiverSD) {
+          absentAliases.push(receiverAlias);
+          absentIDs.push(receiverId);
+        }
+        logger.error(
+          `FHIT could not locate a Structure Definition for ${
+            absentAliases.length > 1
+              ? `${absentAliases[0]}:${absentIDs[0]} or ${absentAliases[1]}:${absentIDs[1]}`
+              : `${absentAliases[0]}:${absentIDs[0]}`
+          }.\n` +
```

Comment:
Also, this is really small, but how would you feel about including spaces after the colons here? I find it easier to read and it aligns with what the YAML configuration file will look like.

```suggestion
        logger.error(
          `FHIT could not locate a Structure Definition for ${
            absentAliases.length > 1
              ? `${absentAliases[0]}: ${absentIDs[0]} or ${absentAliases[1]}: ${absentIDs[1]}`
              : `${absentAliases[0]}: ${absentIDs[0]}`
          }.\n` +
```

Reply:
I _almost_ said something about this in my review, but decided not to say anything (and that it was probably a matter of preference).  But now that @jafeltra has said it... I agree!

Reply:
Updated in commit [8aa5eaf](https://github.com/standardhealth/fhir-matches/pull/10/commits/8aa5eaf626a8f15567ce098e1354516ef975bc90)!